sdk/communication/Azure.Communication.Chat/src/ChatThreadClient.cs

```diff
@@ -343,13 +343,13 @@ public virtual Response DeleteMessage(string messageId, CancellationToken cancel
         /// <param name="participant"> Participant to add to a chat thread. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
-        public virtual async Task<Response> AddParticipantAsync(ChatParticipant participant, CancellationToken cancellationToken = default)
+        public virtual async Task<Response<AddChatParticipantsResult>> AddParticipantAsync(ChatParticipant participant, CancellationToken cancellationToken = default)
```

Comment:
it is a bit weird to me that the method to add one ChatParticipant returns AddChatParticipant**s**Result

Reply:
Btw I don't see these Result changes in Java yet

Reply:
yeah, it is by design, but I agree it is a bit weird. Will create a task to follow up on this before the azure review.
@danielgerlag  is the JAVA sdk using the latest swagger ?